config/locales/fr-CA.yml

```diff
@@ -0,0 +1,50 @@
+---
+fr-CA:
+  collection:
+    contributor_activity:
+      recent_ocr_correction: Correction d'ROC récente
```

Comment:
We need to replace `d'ROC` with `de ROC` since there is no longer a vowel, so there should no contraction of `de` to `d'`.